CMakeLists.txt

```diff
@@ -774,7 +824,17 @@ add_custom_command(
   # NB: COMMENT only works for some CMake-Generators
 )
 
-add_subdirectory(cmake/qemu)
+if(EMU_PLATFORM STREQUAL "qemu")
+	add_subdirectory(cmake/qemu)
+elseif(EMU_PLATFORM STREQUAL "native")
+    add_custom_target(run
```

Comment:
We should be consistent with qemu.

so

add_subdirectory(cmake/native)

Reply:
=> @nashif